.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 package-lock.json
-dist
\ No newline at end of file
+dist
+src/*.js
+src/*.js.map
```

Comment:
Are you emitting js files to `src/*`??

Reply:
Yeah, I have a bunch of plugins in VSCode.  It might even be a native feature?  I'm not sure how they got there to be honest, but the JS and map files were being compiled automatically upon saving.  Perhaps this can be removed and instructions can be added somewhere in the CONTRIBUTING.md.

Reply:
It must be a plugin then because the `tsconfig.json` file specifies the `outDir` as `dist`. Try disabled plugins and see if you still have the problem.